### PR TITLE
Home fixes 1

### PR DESCRIPTION
**Pull request by EricWVGG**
fixes a couple bugs in the donate box and leaderboard UI

### CODE REVIEW

File: src/components/Leaderboard/index.tsx [hunk 1]
```diff
@@ -79,20 +79,23 @@ const Leaderboard = ({ grantId, setAmountRaised }: ILeaderboard) => {
...
   }, [data])
 
-  const sideItem = (
-    <Button outline level={2} onClick={() => setLimit(limit === DEFAULT_LIMIT ? 9999 : DEFAULT_LIMIT)}>
-      {limit === DEFAULT_LIMIT ? 'See All' : 'See Less'}
-    </Button>
-  )
+  const sideItem =
+    donatingUsers && donatingUsers.length > limit ? (
+      <Button outline level={2} onClick={() => setLimit(limit === DEFAULT_LIMIT ? 9999 : DEFAULT_LIMIT)}>
```
Comment (geometrizen) [inline src/components/Leaderboard/index.tsx:84]: just wondering, should `9999` here be hardcoded? and if so, should it be 9999, or 5?

File: src/components/Leaderboard/index.tsx [hunk 2]
```diff
@@ -79,20 +79,23 @@ const Leaderboard = ({ grantId, setAmountRaised }: ILeaderboard) => {
...
+        {limit === DEFAULT_LIMIT ? 'See All' : 'See Less'}
+      </Button>
+    ) : (
+      <></>
+    )
 
-  return !!loading || !data?.Users || !donatingUsers ? (
-    <Spinner />
+  return !donatingUsers ? (
+    <Spinner minHeight="65px" />
```
Comment (geometrizen) [inline src/components/Leaderboard/index.tsx:92]: seems as though this will make the leaderboard less spinny spinny 👍

**Review by geometrizen — COMMENTED** [review]
LGTM, just wondering about the hardcoded 9999

File: src/components/Leaderboard/index.tsx (hunk 1 above)
Comment (EricWVGG) [reply src/components/Leaderboard/index.tsx:84]: it just means "no limit". Maybe we can do it smarter later but right now I just don't care.

**Review by geometrizen — APPROVED** [review]
LGTM

**Pull request merged**